packages/components/popover/src/PopoverContent/PopoverContent.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { AllHTMLAttributes } from 'react';
+import { cx } from 'emotion';
+import { CommonProps } from '@contentful/f36-core';
+import { usePopoverContext } from '../PopoverContext';
+import { Portal } from '@contentful/f36-utils';
+import { getPopoverContentStyles } from './PopoverContent.styles';
+
+interface PopoverContentProps
+  extends AllHTMLAttributes<HTMLDivElement>,
+    CommonProps {}
+
+const PopoverContent = (props: PopoverContentProps, ref) => {
+  const {
+    children,
+    className,
+    testId = 'cf-ui-popover-content',
+    ...otherProps
+  } = props;
+
+  const styles = getPopoverContentStyles();
+
+  const { isOpen, getPopoverProps, usePortal } = usePopoverContext();
+
+  const content = (
+    <div
+      {...getPopoverProps(otherProps, ref)}
+      className={cx(styles.container, className)}
+      data-test-id={testId}
+      tabIndex={-1}
+      role="dialog"
+      // specific attribute to mark that this element is absolute positioned
+      // for internal contentful apps usage
+      data-position-absolute
+    >
+      {children}
+    </div>
+  );
+
+  if (!isOpen) {
+    return null;
+  }
+
+  return usePortal ? <Portal>{content}</Portal> : content;
+};
+
+const _PopoverContent = React.forwardRef(PopoverContent);
+export { _PopoverContent as PopoverContent };
```

Comment:
Because we are doing this in our components, in some places the source code is showing `<_PopoverContent>` (especially in storybook)

in this PR https://github.com/contentful/forma-36/pull/1209 
I'm proposing inverting this names, so basically the one going inside forwardRef is the one with `_` and the one exported has the final name

```suggestion
export const PopoverContent = React.forwardRef(_PopoverContent);
```

Wdyt? this way we avoid the whole `as` thing